release-gap-package

```diff
@@ -268,11 +268,14 @@ esac
 if [ -f makedoc.g ] ; then
     notice "Building GAP package documentation (using makedoc.g)"
     run_gap <<GAPInput
+LogTo("makedoc.log");
 if not IsPackageMarkedForLoading("$PKG", "") then
   SetPackagePath("$PKG", ".");
 fi;
 Read("makedoc.g");
 GAPInput
+    ! grep -E "WARNING: non resolved reference" makedoc.log
+    rm -f makedoc.log
```

Comment:
Shouldn't this also be done for the second run of `makedoc.g`? After all, they happen in different directories, and so it could happen that in one run the references can be resolved, but in the other they can't.